pkg/chrootarchive/archive_test.go

```diff
@@ -99,7 +99,8 @@ func TestChrootUntarWithHugeExcludesList(t *testing.T) {
 	// 65534 entries of 64-byte strings ~= 4MB of environment space which should overflow
 	// on most systems when passed via environment or command line arguments
 	excludes := make([]string, 65534)
-	for i := 0; i < 65534; i++ {
+	var i rune
+	for i = 0; i < 65534; i++ {
```

Comment:
nit: `i` is already 0. Alternatively, `for i := rune(0); …` might be a bit cleaner. But it doesn't really matter, LGTM either way.

Reply:
Ah, yes, could've done that 👍 
Think I'll keep it as-if, as CI is a bit flaky (don't want to be running it multiple times to get green again 😞)